web/src/server/api/routers/generations/db/getAllGenerationsSqlQuery.ts

```diff
@@ -70,19 +70,21 @@ export function getAllGenerationsSqlQuery({
         SELECT
           trace_id,
           observation_id,
-          jsonb_object_agg(name::text, avg_value::double precision) AS scores_avg
+          jsonb_agg(jsonb_build_object('name', name, 'value', avg_value, 'comment', comment)) AS scores_avg
```

Comment:
This now returns the avg_value with all scores - correct? Instead we want to filter by the avg_value and fetch all the scores which belong to the generations. I doubt we can fetch them all in this cte.